### PR TITLE
Add rubygem-apipie-dsl package

### PR DESCRIPTION
**Pull request by ofedoren**
Needed for https://github.com/theforeman/foreman/pull/7537.

### CODE REVIEW

**Comment by xprazak2** [comment]
Looks like the build is passing here...

**Comment by ofedoren** [comment]
@ekohl, rebased. But seems that nothing has changed. Should I re-run scripts and re-create the PR?

**Comment by xprazak2** [comment]
Seems like it needs another rebase...

**Comment by ofedoren** [comment]
@xprazak2, @ekohl, updated. Now tito.props changes are minimal :)

**Comment by ehelms** [comment]
You'll need to add this as well to `comps/comps-foreman-el8.xml` now.

**Comment by ekohl** [comment]
@ehelms We should add that to `add_*_package.sh` and friends or it's bound to be a repeating review comment.

**Comment by ares** [comment]
@ofedoren please add it to el8 definition so this one can get it :-)

**Comment by ofedoren** [comment]
@ares, @ehelms, done :)

File: comps/comps-foreman-el8.xml [hunk 1]
```diff
@@ -164,6 +164,7 @@
       <packagereq type="default">rubygem-apipie-bindings</packagereq>
       <packagereq type="default">rubygem-apipie-params</packagereq>
       <packagereq type="default">rubygem-apipie-rails</packagereq>
+      <packagereq type="default">rubygem-apipie-dsl</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-el8.xml:167]: Mind keeping this list sorted?

**Comment by ofedoren** [comment]
@ekohl, updated. Is that better?

**Comment by tbrisker** [comment]
appears this only got built for el7?

**Comment by tbrisker** [comment]
opened https://github.com/theforeman/foreman-packaging/pull/5111 for el8 build

**Pull request merged**